src/view/loader.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+import Background from '../component/background';
+import { CopyText } from '../component/text';
+import { FormStretcher } from '../component/form';
+import Icon from '../component/icon';
+import { color } from '../component/style';
+
+const styles = StyleSheet.create({
+  copyTxt: {
+    flexDirection: 'row',
+    fontSize: 53,
```

Comment:
We should try to avoid setting hard coded font sizes in the views directly and use predefined font sizes from:
https://github.com/lightninglabs/lightning-app/blob/master/src/component/style.js#L25